projects/gsc/experiments/base.py

```diff
@@ -128,11 +131,12 @@
     ),
 
     # Loss function. See "torch.nn.functional"
-    loss_function=torch.nn.functional.cross_entropy,
+    loss_function=torch.nn.functional.nll_loss,
```

Comment:
This was the main thing to change given our previous discussion. There's also `batch_norm_weight_decay` which now has no effect given [this](https://github.com/numenta/nupic.research/pull/460) PR (To control the batch norm weight decay, you'd need the `ConfigureOptimizerParamGroups` which isn't currently being used by the GSC experiment).

This is all to say, `batch_norm_weight_decay` should be removed from this config as it has no effect on training. As well, is the desired behavior per our discussion (weight decay applied to all params including batchnorm).

Reply:
Minor point: You can increase the size of `val_batch_size`. Right now it's 16, but a larger size could speed up inference.